_posts/2016-03-14-nglview.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Molecular visualization in the Jupyter Notebook with nglview
+---
+
+**[nglview](https://github.com/arose/nglview)** is a Python package that makes it
+easy to visualize molecular systems, including trajectories, directly in
+the [Jupyter Notebook](http://jupyter.org/). The [recent 0.4.0
+release](https://twitter.com/asrmoin/status/701677261980700672) of nglview
+brings a convenient interface for visualizing MDAnalysis `Universe` and
+`AtomGroup` objects directly:
+
+{% highlight python %}
+import MDAnalysis as mda
+import nglview as nv
+from nglview.datafiles import PDB, XTC
+
+u = mda.Universe(PDB, XTC)
+
+protein = u.select_atoms('protein')
+{% endhighlight %}
+
+Although we could use the full `Universe`, selecting a subset of the atoms as
+an `AtomGroup` will give better performance when viewing the trajectory
+interactively. We can feed this directly to the `nglview.show_mdanalysis`
+function to produce the Jupyter widget:
+
+{% highlight python %}
+w = nv.show_mdanalysis(protein)
+w
+{% endhighlight %}
+
+This will give something like this in the notebook itself:
+
+<html lang="en">
```

Comment:
When you look at the served source you get illegal HTML, at least from quick searching it seems to be the case that nested HTML tag or nested body tags are incorrect. 

--
Oliver Beckstein
email: orbeckst@gmail.com


Am Mar 15, 2016 um 10:12 schrieb David Dotson <notifications@github.com>:

> Actually, as far as I can tell from looking around, inserting html in this way is both valid and the way to do it with jekyll.